tests/wpt/metadata/xhr/send-data-sharedarraybuffer.any.js.ini

```diff
@@ -1,5 +1,47 @@
 [send-data-sharedarraybuffer.any.html]
-  expected: CRASH
+  [sending a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Int8Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Uint8Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Uint8ClampedArray backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Int16Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Uint16Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Int32Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Uint32Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Float32Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Float64Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a DataView backed by a SharedArrayBuffer]
+    expected: FAIL
 
 [send-data-sharedarraybuffer.any.worker.html]
-  expected: CRASH
+  [sending a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Int8Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Uint8Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Uint8ClampedArray backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Int16Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Uint16Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Int32Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Uint32Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Float32Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a Float64Array backed by a SharedArrayBuffer]
+    expected: FAIL
+  [sending a DataView backed by a SharedArrayBuffer]
+    expected: FAIL
```

Comment:
nit: do we always want to have a newline at the end of `.ini` files? If only to not have github show the warning in the diff?

Reply:
I'm not convinced it's a useful thing to care about.

Reply:
ok!